### PR TITLE
Makes a group jaunt spell

### PR DESCRIPTION
**Pull request by clusterfack**
1. Adds invocation to jaunt and group jaunt
2. Creates an adminbus group jaunt spell, gives everyone jaunting for double the time of regular jaunt with a double cooldown as well in a 3 tile range
3. Group jaunting makes an image so all jaunters can see each other
4. Group jaunt adminbus only and maybe will be placed on a secret map or away mission or vault some day so it requires no robes

### CODE REVIEW

**Comment by clusterfack** [comment]
Could use some new enter/exit animations and a new hud sprite to be honest

File: code/modules/spells/targeted/ethereal_jaunt.dm [hunk 1]
```diff
@@ -76,6 +90,18 @@
 	target.alphas -= "etheral_jaunt"
 	target.handle_alpha()
 
+/spell/targeted/ethereal_jaunt/jauntgroup
+	name = "Group Jaunt"
+	desc = "This spell allows all people within range to be jaunted along with the user"
+	spell_flags = Z2NOCAST | INCLUDEUSER //Adminbus spell, so we'll exclude robe requirement for ease of use
+
+	invocation = "T'lcn Oc'lip"
```
Comment (9600bauds) [inline code/modules/spells/targeted/ethereal_jaunt.dm]: Nice

Comment (Intigracy) [inline code/modules/spells/targeted/ethereal_jaunt.dm]: I get the n Oc'lip but what's with the T'lc

Comment (9600bauds) [inline code/modules/spells/targeted/ethereal_jaunt.dm]: Tender love and care

Comment (clusterfack) [inline code/modules/spells/targeted/ethereal_jaunt.dm]: It was supposed to be tcl

**Comment by PJB3005** [comment]
``` DM
code/modules/spells/targeted/ethereal_jaunt.dm:31:error: target.incoporeal_move: undefined var
code/modules/spells/targeted/ethereal_jaunt.dm:31:error: INCOPOREAL_ETHEREAL: undefined var
```

**Comment by clusterfack** [comment]
There's just a small problem where when you incorporeal move the image I'm using so you can see other jaunters isn't updating its direction, I'm not sure how to do that really, perhaps an on_moved event?

**Comment by clusterfack** [comment]
Okay, now it uses an on_moved event

File: code/modules/spells/targeted/ethereal_jaunt.dm [hunk 2]
```diff
@@ -76,6 +83,43 @@
...
+						target.client.images -= jaunts[A]
+
+/spell/targeted/ethereal_jaunt/jauntgroup
+	name = "Group Jaunt"
+	desc = "This spell allows all people within range to be jaunted along with the user"
+	hud_state = "group_jaunt"
+
+	spell_flags = Z2NOCAST | INCLUDEUSER //Adminbus spell, so we'll exclude robe requirement for ease of use
+
+	invocation = "Tc'ln Oc'lip"
```
Comment (NigglyWiggly) [inline code/modules/spells/targeted/ethereal_jaunt.dm:116]: Heh

**Comment by DrCelt** [comment]
If you merge this using client images, I can make this work with planes in #11340

**Comment by clusterfack** [comment]
No, you can't.

**Comment by DrCelt** [comment]
@clusterfack yes, I can.

**Pull request merged**